examples/custom-sidebar/index.js

```diff
@@ -26,7 +26,7 @@ app.get('/', function (req, res) {
 app.post('/custom-sidebar', verifyMiddleware, function (req, res) {
   setTimeout(function () {
     res.render('sidebar.html', {
-      details: JSON.stringify(req.body, null, '  ')
+      details: JSON.stringify(req.data, null, '  ')
```

Comment:
This is also from the body-parser middleware in express, and should be `req.body`.